tools/sklearn/fitted_model_eval.xml

```diff
@@ -1,11 +1,11 @@
-<tool id="sklearn_fitted_model_eval" name="Evaluate a Fitted Model" version="@VERSION@">
+<tool id="sklearn_fitted_model_eval" name="Evaluate a Fitted Model" version="@VERSION@" profile="20.05">
```

Comment:
Any particular reason for targeting profile '20.05'?

Reply:
we can also set 20.01, but we should set one. Galaxy will treat $HOME etc differently with a set profile.